augur/filter/include_exclude_rules.py

```diff
@@ -1,9 +1,10 @@
-import ast
 import json
 import operator
 import re
 import numpy as np
 import pandas as pd
+# FIXME: check that this import works with all pandas versions supported in setup.py
+from pandas.core.computation.parsing import tokenize_string, BACKTICK_QUOTED_STRING
```

Comment:
I'm having trouble installing `pandas==1.0.0` on both my local machine and a GitHub Codespace (failed building wheel for an older version of numpy), but [source code](https://github.com/pandas-dev/pandas/blob/v1.0.0/pandas/core/computation/parsing.py) indicates that both of these imports should still work. The [commit](https://github.com/pandas-dev/pandas/commit/fffb9784559ed9797235a4e79692bc8205371478) that added backtick quoting in Pandas says the feature was added in version 0.25.0.